.mailmap

```diff
@@ -1616,6 +1616,7 @@ richierichrawr <richierichrawr@users.noreply.github.com>
 rimibis <33387803+rimibis@users.noreply.github.com>
 risubaba <risubhjain1010@gmail.com>
 ritikBhandari <ritikbhandari68@gmail.com>
+rrodenbusch <rrodenbusch@gmail.com>
```

Comment:
Usually the idea is to add a .mailmap entry like:
```
Richard Rodenbusch <rrodenbusch@gmail.com> rrodenbusch <rrodenbusch@gmail.com>
```
The first name and email address is what will eventually go in the AUTHORS file. The second is what is recorded in the commit metadata. This line in the .mailmap file associates the author name with the corresponding commits.

There are a number of entries not like that in the .mailmap file that mostly come from people who have misunderstood what these entries are intended to do.

Reply:
I will update my entry.  The entry in the documentation for this process mentions the dual entry format after the step to commit the change to .mailmap.   Perhaps we should re-order that?

Reply:
Maybe. I'm not sure which part you are referring to. Is it this:
https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#add-your-name-and-email-address-to-the-mailmap-file

Reply:
Yes.  That is the document I was following.  I'd tweak the document so it shows the two entry option in the initial section, then  links to the section below with the more detailed description on the entries. I'm attaching changes here so you can look at it before I add it to the PR.

### _first section_

The first time you make a pull request you will need to add your name and email address to the .mailmap file by adding a line like

```
Joe Bloggs <joe@bloggs.com>  joeb <joe@bloggs.com>
```

This line in the .mailmap file associates the author name with the corresponding commits. The first name and email address is what will eventually go in the AUTHORS file. The second entry is what is recorded in the commit metadata. (see [Mapping user names to AUTHORS file entry](#mailmap-mapping-names))

The commit metadata name and email should exactly match the name and email that you have configured with git before making the commits (see "Configure git settings" above). The `bin/mailmap_check.py` script can check that this has been done correctly. 


### _then_ _below_

(mailmap-mapping-names)=
### Mapping user names to AUTHORS file entry
Sometimes a commit will be made with an incorrect name or email address or an author will make multiple commits with different names and email addresses or an author wishes to use a proper name that differs from their github name. In this case a line should be added to the .mailmap file where the first name and email address is what should be recorded in the AUTHORS file and the others are the name and email address that was incorrectly used in the other commits.


### _the updated md_
[workflow-process.md](https://github.com/sympy/sympy/files/15168077/workflow-process.md)

Reply:
Yes, maybe that is better.

The premise of the way it is written currently is that you will configure git to have the right name and email address first:
```console
$ git config user.name
Oscar Benjamin
```
It seems that it refers to "configure git settings" that is not present in the document though.

The two entry version is only needed if the name configured in git is not the name that should be in the authors file. It seems to be very common though that the name and/or email in the commits is not the right one for the authors file so it is probably more common to need the two entry version. I agree that
```
Joe Bloggs <joe@bloggs.com>  joeb <joe@bloggs.com>
```
is probably the best thing to suggest by default.

Reply:
I found 'configure git settings'  in dev-setup.md and added the links.   Do I add these changes be added to this PR or should I generate a new one?

[workflow-process.md](https://github.com/sympy/sympy/files/15168941/workflow-process.md)

[dev-setup.md](https://github.com/sympy/sympy/files/15168959/dev-setup.md)

Reply:
I'd say go with a new PR for the workflow changes.

Reply:
In the new PR you won't have the problems because you are already in the .mailmap file (unless you change your git config...).